src/index.tsx

```diff
@@ -5,13 +5,13 @@ const contextEnabled = Object.prototype.hasOwnProperty.call(React, "createContex
 const hooksEnabled = Object.prototype.hasOwnProperty.call(React, "useMemo") && Object.prototype.hasOwnProperty.call(React, "useCallback");
 
 export interface IPrintContextProps {
-    handlePrint: () => void,
+    handlePrint: (event: React.MouseEvent, content?: (() => React.ReactInstance | null)) => void,
```

Comment:
I realized one last thing: it's possible that some users pass this function to other type of event callbacks, and this type would break there. For example:

![Screenshot 2024-02-07 001840](https://github.com/gregnb/react-to-print/assets/1287375/2c0122c1-5039-44c2-a2e4-0f3f280b37f8)

Can we change this to just be `unknown` for now since we don't need it?
```suggestion
    handlePrint: (event: unknown, content?: (() => React.ReactInstance | null)) => void,
```

Reply:
I think we'll need this change in all the places we've typed this, to maintain backwards compatibility

Reply:
All done now @MatthewHerbst . Thanks a lot